services/scrape-test/insert_youtube_url.py

```diff
@@ -0,0 +1,49 @@
+from google.cloud import bigquery
+from typing import Dict, Any
+import datetime
+import logging
+
+logger = logging.getLogger(__name__)
+
+
+def serialize_row(row: Dict[str, Any]) -> Dict[str, Any]:
```

Comment:
This function name is misleading. It's not really doing serialization. it just reformats date. Also, maybe consider create data transfer object from it here using for example `pydantic` or `@dataclass`